packages/appstore/src/components/app-launcher/app-launcher.scss

```diff
@@ -13,8 +13,20 @@
     }
 
     &__button {
-        height: fit-content;
         margin-left: auto;
+        display: flex;
```

Comment:
why we need these changes in this file since we are not using the Appluncher component in this PR?

Reply:
Revert these changes , It was implemented earlier since we started with this app launcher in the beginning.